linode/manager
# fix: [M3-7197] - "Support Ticket" button in network tab not working properly

## Description 📝
PR fixes "Support Ticket" button in network tab.

## Changes  🔄
List any change relevant to the reviewer.
- Used `SupportLink` component. 
- Created `ExplainerCopy` and referenced in `AddIPDrawer`.

## Target release date 🗓️
Please specify a release date to guarantee timely review of this PR. If exact date is not known, please approximate and update it as needed.

## Preview 📷
![image](https://github.com/user-attachments/assets/94589c90-5ac1-4a77-af35-bcfd77145322)


## How to test 🧪

### Verification steps
(How to verify changes)
- Go to the Cloud Manager
- Click on my Linode and go to the Network tab
- Click on the Add an IP Address option then a pop-up comes up for adding an IP address, then you can see a support ticket link once you click on Public IP.

## As an Author I have considered 🤔

*Check all that apply*
